PennMobile/src/main/java/com/pennapps/labs/pennmobile/DiningInfoFragment.kt

```diff
@@ -0,0 +1,176 @@
+package com.pennapps.labs.pennmobile
+
+import android.content.Context
+import android.graphics.Color
+import android.net.ConnectivityManager
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.RelativeLayout
+import android.widget.TextView
+import android.widget.Toast
+import androidx.fragment.app.Fragment
+import com.google.android.gms.maps.CameraUpdateFactory
+import com.google.android.gms.maps.GoogleMap
+import com.google.android.gms.maps.OnMapReadyCallback
+import com.google.android.gms.maps.SupportMapFragment
+import com.google.android.gms.maps.model.LatLng
+import com.google.android.gms.maps.model.MarkerOptions
+import com.pennapps.labs.pennmobile.api.Labs
+import com.pennapps.labs.pennmobile.classes.DiningHall
+import com.pennapps.labs.pennmobile.classes.VenueInterval
+import kotlinx.android.synthetic.main.fragment_dining_info.view.*
+import org.joda.time.format.DateTimeFormat
+import rx.android.schedulers.AndroidSchedulers
+
+/**
+ * Created by Lily on 11/13/2015.
+ * Fragment for Dining information (hours, map)
+ */
+class DiningInfoFragment : Fragment(), OnMapReadyCallback {
```

Comment:
What is this class used for again? We probably don't need the map stuff but we can take it out later